pkgs/development/compilers/ghc/8.4.4.nix

```diff
@@ -109,7 +109,11 @@ stdenv.mkDerivation (rec {
       name = "D4388.diff";
       sha256 = "0w6sdcvnqjlnlzpvnzw20b80v150ijjyjvs9548ildc1928j0w7s";
     })
-    ++ stdenv.lib.optional stdenv.isDarwin ./backport-dylib-command-size-limit.patch;
+    ++ stdenv.lib.optional stdenv.isDarwin ./backport-dylib-command-size-limit.patch
+    ++ stdenv.lib.optional (targetPlatform.isAarch32 || targetPlatform.isAarch64) (fetchpatch {
+      url = "https://git.haskell.org/ghc.git/patch/d8495549ba9d194815c2d0eaee6797fc7c00756a";
+      sha256 = "1czx12qcl088vjn7mqxvyja4b2ia2n09c28br8c777fd0xk069pm";
```

Comment:
Just a heads up that this doesn't appear to be a correct hash for `fetchpatch`, I'm getting

```shell
nix-build -E - <<EOF                                                                                   
with import ./. {};
fetchpatch {
  url = "https://git.haskell.org/ghc.git/patch/d8495549ba9d194815c2d0eaee6797fc7c00756a";
  sha256 = "1czx12qcl088vjn7mqxvyja4b2ia2n09c28br8c777fd0xk069pm";
}
EOF
these derivations will be built:
  /nix/store/87nibcpvm6kgg6yda6b8nnbhngbb902k-d8495549ba9d194815c2d0eaee6797fc7c00756a.drv
building '/nix/store/87nibcpvm6kgg6yda6b8nnbhngbb902k-d8495549ba9d194815c2d0eaee6797fc7c00756a.drv'...

trying https://git.haskell.org/ghc.git/patch/d8495549ba9d194815c2d0eaee6797fc7c00756a
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  8546    0  8546    0     0   8154      0 --:--:--  0:00:01 --:--:--  8154
fixed-output derivation produced path '/nix/store/garw8dzcc39w8dgzlrvjrphr3vpwfr1n-d8495549ba9d194815c2d0eaee6797fc7c00756a' with sha256 hash '1yjcma507c609bcim4rnxq0gaj2dg4d001jklmbpbqpzqzxkn5sz' instead of the expected hash '1czx12qcl088vjn7mqxvyja4b2ia2n09c28br8c777fd0xk069pm'
error: build of '/nix/store/87nibcpvm6kgg6yda6b8nnbhngbb902k-d8495549ba9d194815c2d0eaee6797fc7c00756a.drv' failed
```

Same on Hydra - https://hydra.nixos.org/build/85456305/nixlog/77

(It would appear to be valid for `fetchurl` though).

Reply:
hmm, seems to have been merged anyway, I guess this isn't a problem...